pkgs/games/heroic/default.nix

```diff
@@ -1,39 +1,69 @@
-{ lib, fetchurl, appimageTools }:
-
-let
-  pname = "heroic";
-  version = "2.2.6";
-  name = "${pname}-${version}";
-  src = fetchurl {
-    url = "https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v${version}/Heroic-${version}.AppImage";
-    sha256 = "sha256-kL30/G4DpDPwGN7PvbWest7TcgL4Rd1c2OM4nRCT3bg=";
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, makeWrapper
+, electron
+
+, autoPatchelfHook
+, stdenv
+, zlib
+}:
+
+mkYarnPackage rec {
+  pname = "heroic-original";
```

Comment:
```suggestion
  pname = "heroic-unwrapped";
```

Reply:
I'd like a second opinion on this change.  I went with the name `heroic-original` to be consistent with Lutris, which serves a similar purpose to Heroic, and they are now structured similarly.  I have no preference myself, but I want to be following current conventions in nixpkgs.

Reply:
I threw this review because the name on the `all-packages.nix` is `heroic-unwrapped`, so it is more to keep it with the same name. You can change it to `heroic-original` though, as Lutris does the same, but it is mostly to keep the same name between the `default.nix` and the `all-packages.nix` files.

Reply:
I did notice that inconsistency with Lutris, so it would not surprise me if a change to the naming of the unwrapped Lutris package comes out of this review.

Reply:
In this case, I would favor naming it as `unwrapped` then, also because Lutris is the only case where it is named as original, but as you said, it uses `unwrapped` on the `all-packages.nix` file.

Reply:
Yes, using ``unwrapped`` here would be the standard way I would choose, too.